modes/debug_mode/debug_plot_page.py

```diff
@@ -89,18 +89,12 @@ def __init__(self, parent: Frame, model: Model):
         # placing the canvas on the Tkinter window
         self.canvas.get_tk_widget().grid(row=0, column=0, sticky="s")
 
-        self.update_pinned_data()
-
     def enter_button_pressed(self):
-        self.current_time_index += 1 if self.current_time_index < len(self.time_presets) - 1 else 0
+        self.current_time_index = self.current_time_index + 1 if self.current_time_index < len(self.time_presets) - 1 else 0
```

Comment:
```suggestion
        self.current_time_index = self.current_time_index + (1 if self.current_time_index < len(self.time_presets) - 1 else 0)
```
if this is what you're going for then the parentheses are necessary because otherwise this could set it to 0 depending on order of operations

Reply:
I want to set it to 0